src/bundle/Resources/views/content/content_view_fields.html.twig

```diff
@@ -19,7 +19,9 @@
                                 <div class="ez-content-field">
                                     <p class="ez-content-field-name">{{ fieldDefinition.name }}:</p>
                                     <div class="ez-content-field-value">
-                                        {% if ez_is_field_empty(content, fieldDefinition.identifier) %}
+                                        {% if ez_is_field_empty(content, fieldDefinition.identifier) and fieldDefinition.fieldTypeIdentifier is same as('ezboolean') %}
+                                            <em>{{ 'ezboolean.no'|trans|desc('No') }}</em>
```

Comment:
I'm not sure if this is the way. I think in the case of ezboolean the field should be rendered even if it's empty.